Common/Securities/SecurityService.cs

```diff
@@ -78,7 +78,15 @@ public class SecurityService : ISecurityService
                 throw new ArgumentException($"Symbol can't be found in the Symbol Properties Database: {symbol.Value}");
             }
 
-            var symbolProperties = _symbolPropertiesDatabase.GetSymbolProperties(symbol.ID.Market, symbol, symbol.ID.SecurityType, defaultQuoteCurrency);
+            // We want to redirect the Symbol Properties Database lookup for future options -> futures,
+            // since future options share the same symbol properties as their underlying future.
+            var hasUnderlyingFuture = symbol.SecurityType == SecurityType.Option && symbol.Underlying.SecurityType == SecurityType.Future;
+            var symbolProperties = _symbolPropertiesDatabase.GetSymbolProperties(
+                hasUnderlyingFuture ? symbol.Underlying.ID.Market : symbol.ID.Market,
+                hasUnderlyingFuture ? symbol.Underlying : symbol,
+                hasUnderlyingFuture ? symbol.Underlying.ID.SecurityType : symbol.ID.SecurityType, 
+                defaultQuoteCurrency);
```

Comment:
Will this also set the other SPDB properties like `minimum_price_variation` / `lot_size` will it be correct for the FOP to use the futures one too?

Reply:
I haven't seen any contract yet with diverging values, but I think we should review some contracts to make sure this behavior is correct

Reply:
Found an example of diverging values: the future and futures option contract for ZB has differing minimum price fluctuations:

https://www.cmegroup.com/trading/interest-rates/us-treasury/30-year-us-treasury-bond_contract_specifications.html
https://www.cmegroup.com/trading/interest-rates/us-treasury/30-year-us-treasury-bond_contractSpecs_options.html